internal/http/server.go

```diff
@@ -23,7 +23,7 @@ import (
 
 const (
 	ModuleV1Prefix = "/v1/modules"
```

Comment:
I mean add a trailing slash here as well.

Reply:
Ah no I didn't do that because in TFC the modules prefix reads like:
"modules.v1":"/api/registry/v1/modules/"
And in OTF it reads like:
"modules.v1":"/v1/modules"

The change is not as obvious. I'm not sure that will affect anything.

Reply:
https://github.com/dahlke/terrasnek/blob/a3bb4d126f21718d73b7563343131c590c736c07/terrasnek/endpoint.py#L37 removes the trailing character, which in the case of OTF would be the `s` in `/v1/modules`.

Reply:
Ah right... ok will add the trailing / only.

Reply:
Ok added the trailing / form modules. I will add a better handling in terrasnek as well.

Reply:
Added the fix for modules.v1 in terrasnek as well.